packages/server/events-processing-platform/notifications/postmark.go

```diff
@@ -46,14 +48,16 @@ func NewPostmarkProvider(log logger.Logger, serverToken string) *PostmarkProvide
 	}
 }
 
-func (np *PostmarkProvider) SendNotification(ctx context.Context, postmarkEmail PostmarkEmail) error {
+func (np *PostmarkProvider) SendNotification(ctx context.Context, postmarkEmail PostmarkEmail, span *opentracing.Span) error {
```

Comment:
opentracing.Span is interface no need to declare it as pointer
and use it without * / &